lib/backend/k8s/converter/workload.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
```

Comment:
Probably want to call this `workloadendpoint.go`

Reply:
This will go away when I rebase against master, consumed by Rob's refactoring of the same files.